### DISCUSSION TITLE
[4.4.7]Bottom sheet reappears even after its closed

### DISCUSSION THREAD

**Discussion by davepaiva** [gorhom/react-native-bottom-sheet#1472] [Q&A]
I upgraded the package to 4.4.7 hoping that it will fix this issue as stated in the release notes. but the issue is still happening. Any idea how to fix this ?

**Comment by clemensmol** [comment]
I encountered the same problem. Do you happen to use the "onAnimate" prop? I no longer use it as a temporary workaround, which resolved this issue for me.

**Reply by davepaiva** [reply]
thanks for the reply, but I have not been using the `onAnimate` prop.
